pgmpy/tests/test_factors/test_Factor.py

```diff
@@ -240,6 +360,15 @@ def test_factor_divide_truediv(self):
         phi3 = Factor(['x1', 'x2'], [2, 2], [1, 2, 1, 2])
         self.assertEqual(phi3, div)
 
+        var1 = ('x2', 1)
+        var2 = frozenset(['x1', 'x2'])
+        phi1 = Factor([var1, var2], [3, 2], [3, 2, 4, 5, 9, 8])
+        phi2 = Factor([var2], [2], [3, 6])
```

Comment:
@abhi95 You can put this all in the `setUp` method. Since you are defining the same factors in each test.